Marlin/src/feature/pause.h

```diff
@@ -139,4 +137,4 @@ bool unload_filament(
 
   constexpr uint8_t did_pause_print = 0;
 
-#endif // !ADVANCED_PAUSE_FEATURE
+#endif // ADVANCED_PAUSE_FEATURE
```

Comment:
I found some other instances of this comment pattern for `#else/#endif` blocks, and I believe that the old style (with the !) was intentional.

```suggestion
#endif // !ADVANCED_PAUSE_FEATURE
```

Reply:
There is actually an example of this in the bottom example of the Marlin coding standards
https://marlinfw.org/docs/development/coding_standards.html